### PR TITLE
fix: don't append Shift modifier text twice to accelerators

### LINKED ISSUE #15392: Accelerator "Shift+Backspace" show as "Shift+Shift+Backspace" in menu
* Output of `node_modules/.bin/electron --version`: v3.0.6
* Operating System (Platform and Version): Windows 10 (October Update)

**Expected Behavior**
When using the accelerator "Shift+Backspace" in a `Menu`, the accelerator should be displayed as "Shift+Backspace"

**Actual behavior**
"Shift+Shift+Backspace" is displayed as accelerator in the menu.

**To Reproduce**
See: https://github.com/torjusb/electron-accelerator-bug

**Screenshots**
accelerator

**Additional info**
This only appears to affect "Shift+Backspace" from limited testing. Accelerators such as "Alt+Backspace", "Shift+M" etc. appears to work fine.

### PR DESCRIPTION
**Pull request by brenca**
#### Description of Change

Fixes #15392. For some reason our patch applied a `Shift` modifier to a whole bunch of accelerator characters (like `Backspace`, but `Tab` and `Enter` were also affected), my best guess is that those used to be broken with `Shift` but now are working properly in `chromium`.

I'll backport this manually, trop couldn't deal with `4-0-x`.
#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

#### Release Notes

Notes: fixed some accelerators having `Shift` appended to them twice

### CODE REVIEW

File: patches/common/chromium/accelerator.patch [hunk 1]
```diff
@@ -19,17 +19,17 @@ index 7e55ef366ac8320f730cdcb268453b1fa2710887..c3fb98b426cd7c12f66eaaf358f4ff18
 @@ -21,9 +22,7 @@
  #include <windows.h>
  #endif
- 
+
```
Comment (nornagon) [inline patches/common/chromium/accelerator.patch]: those spaces are important, leave them in please! this patch should be exactly as exported by `git-export-patches`.

Comment (brenca) [reply patches/common/chromium/accelerator.patch]: I knew there was something up with those spaces, gonna fix asap

**Comment by ckerr** [comment]
@nornagon thoughts on the revised PR?

**Comment by brenca** [comment]
@nornagon ask, and thou shall receive 🙇

File: atom/browser/api/atom_api_menu.cc [hunk 2]
```diff
@@ -155,6 +155,13 @@ base::string16 Menu::GetSublabelAt(int index) const {
   return model_->GetSublabelAt(index);
 }
 
+base::string16 Menu::GetAcceleratorTextAt(int index) const {
+  auto* accelerator = new ui::Accelerator();
```
[the remark this answers is not in the document]
Comment (nornagon) [reply atom/browser/api/atom_api_menu.cc]: oooh, good catch! this should probably be on the stack.
```suggestion
  ui::Accelerator accelerator;
  model_->GetAcceleratorAtWithParams(index, true, &accelerator);
```

[the remark this answers is not in the document]
Comment (brenca) [reply atom/browser/api/atom_api_menu.cc]: Ah, yeah, that's just a temporary that gets filled with the details, gonna make that a `std::unique_ptr`

[the remark this answers is not in the document]
Comment (brenca) [reply atom/browser/api/atom_api_menu.cc]: Done! Thanks for catching this

**Pull request merged**